documentation/modules/overview/con-configuration-points-mirrormaker.adoc

```diff
@@ -5,9 +5,17 @@
 [id="configuration-points-topic_{context}"]
 = Kafka MirrorMaker configuration
 
-To set up MirrorMaker, a source and destination Kafka cluster must be running.
+To set up MirrorMaker, a source and target (destination) Kafka cluster must be running.
 
-MirrorMaker requires:
+You can use {ProductName} with MirrorMaker or MirrorMaker 2.0.
+
+[discrete]
+== MirrorMaker
+
+MirrorMaker uses producers and consumers to replicate data across clusters.
+MirrorMaker 2.0 is based on the Kafka Connect framework, _connectors_ managing the transfer of data between clusters.
```

Comment:
This sentence looks a bit weird here, because the chapter seems to be about Mirror Maker 1. Is it here intentionally?

Reply:
Oh! Not sure how that ended up there. Thanks Jakub.
I've moved it back to where it was -- under the _MirrorMaker 2.0_ sub-section.